### PR TITLE
Added docs for ListItemModel and ListSortFieldSelectorModel

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-common?_sp.spa=pr-docs](https://developer.blackbaud.com/skyux-list-builder-common?_sp.spa=pr-docs)

File: src/app/docs/list-builder-common/grid-docs.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<sky-docs-demo-page
+  moduleSourceCodePath="src/app/public/state/"
+  pageTitle="List builder common"
```
Comment (blackbaud-johnly) [inline src/app/docs/list-builder-common/grid-docs.component.html]:
```suggestion
  pageTitle="List classes"
```

File: src/app/docs/list-builder-common/grid-docs.component.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+>
+  <sky-docs-demo-page-summary>
+    Shared classes for the list component.
```
Comment (blackbaud-johnly) [inline src/app/docs/list-builder-common/grid-docs.component.html]:
```suggestion
    Shared classes for list components.
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-common?_sp.spa=pr-docs](https://developer.blackbaud.com/skyux-list-builder-common?_sp.spa=pr-docs)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**